scripts/e2e-tests.sh

```diff
@@ -147,11 +151,12 @@ function create_test_cluster() {
   set -o errexit
   set -o pipefail
 
-  header "Creating test cluster"
+  header "Creating test cluster with --min-nodes=${E2E_MIN_CLUSTER_NODES} --max-nodes=${E2E_MAX_CLUSTER_NODES}"
```

Comment:
move `--min-node` and `max-nodes` out of "header" might be better

Reply:
Agreed. Holding for the suggestion.

/hold

Reply:
Done

Reply:
yeah that's exactly what I meant, something like this:
`echo "args:\n\t--min-nodes=${E2E_MIN_CLUSTER_NODES}\n\t--max-nodes=${E2E_MAX_CLUSTER_NODES}"`

Reply:
I suggest a human-readable logging like

```echo "Cluster will have a minimum of ${E2E_MIN_CLUSTER_NODES} and a maximum of ${E2E_MIN_CLUSTER_NODES} nodes"```

Reply:
sounds good to me

Reply:
Done